util.c

```diff
@@ -5555,3 +5555,25 @@ void util_setup_coredumps()
   config.core_setup = 1;
 }
 
+
+/*
+ * Write s390 config values into /boot/zipl/active_devices.txt
+ * (see bsc#1095062).
+ */
+void util_write_active_devices(char *format, ...)
+{
+  const char *file_name = "/boot/zipl/active_devices.txt";
```

Comment:
I know that this is the path quoted in the bug report, but shouldn't it be  `/mnt/boot/...` instead?

Reply:
No. it's in the inst-sys.

Reply:
AFAIK not, the file is copied to `/mnt` later by YaST...

Reply:
no, it is in insts-sys and we copy it from insts-sys. /mnt should not be ready in this time, not?

Reply:
See https://github.com/yast/yast-installation/pull/705/files